dali/operators/reader/numpy_reader_op.cc

```diff
@@ -55,11 +55,14 @@ DALI_SCHEMA(NumpyReader)
   .AddArg("file_root",
       R"code(Path to a directory containing data files.
 `NumpyReader` supports flat directory structure. `file_root` directory should contain
-directories with numpy files in them.)code",
-      DALI_STRING)
+directories with numpy files in them.)code", DALI_STRING)
   .AddOptionalArg("file_filter",
       R"code(If specified, the string will be interpreted as glob string to filter
 the list of files in the sub-directories of `file_root`.)code", "*.npy")
+  .AddOptionalArg("file_list",
+      R"code(Path to a text file containing rows of ``filename label`` pairs, where the filenames are
+relative to ``file_root``. If left empty, ``file_root`` is traversed for subdirectories (only those at one level deep from
+``file_root``).)code", std::string())
```

Comment:
std::string() or DALI_STRING, @mzient ?

Reply:
std::string(), because the default is an empty string. It detects the type based on what is put there.